packages/legend-query-builder/src/components/result/QueryBuilderResultPanel.tsx

```diff
@@ -419,13 +421,54 @@ export const QueryBuilderResultPanel = observer(
                 Running Query...
               </div>
             )}
-
-            <div
-              data-testid={QUERY_BUILDER_TEST_ID.QUERY_BUILDER_RESULT_ANALYTICS}
-              className="query-builder__result__analytics"
-            >
-              {resultDescription ?? ''}
-            </div>
+            {resultDescription && resultState.executionTraceID && (
+              <div
+                data-testid={
+                  QUERY_BUILDER_TEST_ID.QUERY_BUILDER_RESULT_ANALYTICS
+                }
+                className="query-builder__result__analytics"
+              >
+                <QueryBuilderBaseInfoTooltip
+                  title="Execution Result Analytics"
+                  data={[
+                    {
+                      label: 'Trace',
+                      value: 'available here',
+                      actionButton: (
+                        <div className="query-builder__tooltip__item__action">
+                          <button
+                            onClick={() => {
```

Comment:
Should we disable this button and add a title if there's no base url configured?
```suggestion
                            disabled={!queryBuilderState.config?.zipkinTraceBaseURL}
                            title={queryBuilderState.config?.zipkinTraceBaseURL ? '' : 'No zipkin trace URL configured'}
                            onClick={() => {
```